src/cpu.cpp

```diff
@@ -16,15 +16,35 @@
 
 /* rdtsc for MSC_VER, uses simple inline assembly, or _rdtsc
  * from external win64.asm because VS2005 does not support inline assembly */
-#if defined(_MSC_VER) && (defined(_M_IX86) || defined(_M_X64)) && !defined(RDTSC_AVAILABLE)
+#if defined(_MSC_VER) && !defined(RDTSC_AVAILABLE)
```

Comment:
Can't comment on commit message, but I would add for which platforms you added arm64, as this is a bit too generic. Something like:

`Feature: Add support for rdtsc for arm64 on Windows and macOS`
?

Reply:
It's also adding it for Linux too (gcc 10 or newer only). I'll amend the comment in the file as suggested above, but I think the generic commit message is probably OK.

Reply:
I tell a lie... __has_builtin is in gcc 10, but __builtin_readcyclecounter is not. However, the cross-platform comment still applies if you build with clang on Linux. Amended the message (again).